### PR TITLE
feat: Add license utility to chectl to validate and add license header to codebase

### PR DESCRIPTION
**Pull request by flacatus**
### What does this PR do?
This PR add a new automation to chectl to check and add license headers to codebase. Also this PR fix all incorrect license headers.

Refference issue: https://github.com/eclipse/che/issues/19647
### How to test this PR?

Download current branch and execute:
For check release:
```bash
/bin/bash .ci/check-license.sh --check-license
```
To add new license header to a file which doesn't contain a license:
```bash
/bin/bash .ci/check-license.sh --add-license
```
### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by benoitf — COMMENTED** [review]
why not using eslint https://www.npmjs.com/package/eslint-plugin-header ?

IMHO typescript projects should use eslint plug-ins so it's handled in IDE as well

**Comment by mmorhun** [comment]
I have another question to discuss. This PR forces all headers to have date `2019-2021`. However, some files haven't been changed since, say, 2020 and some were created in 2021. To my mind, it is not right to force dates in license headers, as they should describe real state of the files.

**Comment by tolusha** [comment]
It isn't a problem to update year.
I used to do it in old che repository every year.

File: .ci/oci_chectl_olm.sh [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#!/bin/bash
```
Comment (AndrienkoAleksandr) [inline .ci/oci_chectl_olm.sh:1]: Maybe for shell script we have to save #!/bin/bash ? I guess they will be broken without this directive...

File: ambient.d.ts [hunk 2]
```diff
@@ -1 +1,14 @@
+/**
+ *
```
Comment (tolusha) [inline ambient.d.ts:2]: Extra empty line

File: .ci/oci_chectl_olm.sh (hunk 1 above)
Comment (flacatus) [reply .ci/oci_chectl_olm.sh:1]: My mistake

**Comment by flacatus** [comment]
Closing in favor of https://github.com/che-incubator/chectl/pull/1280

**Pull request closed without merging**